hibernate-core/src/test/java/org/hibernate/orm/test/dialect/resolver/DiscoveredDialectResolverTests.java

```diff
@@ -40,6 +42,7 @@
 )
 @DomainModel
 @SessionFactory
+@SkipForDialect(dialectClass = AltibaseDialect.class, reason = "bypass test because AltibaseDialect is Community Dialect")
```

Comment:
Might be better to change this to `@RequiresDialect(H2Dialect.class)`

Reply:
Yeah. I'm gonna use @RequiresDialect annotation